### PR TITLE
Update .gitlab-ci.yml

### PR DESCRIPTION
**Pull request by mordamax**

### CODE REVIEW

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -84,114 +84,114 @@ variables:
...
-  stage:                           test
-  <<:                              *docker-env
-  <<:                              *common-refs
-  # this is an artificial job dependency, for pipeline optimization using GitLab's DAGs
-  needs:
-    - job:                         check-rustdoc
-      artifacts:                   false
-  script:
-    - time cargo check --all --benches
+#test-linux-stable:
```
Comment (bkontur) [inline .gitlab-ci.yml:87]: @mordamax 
What is the reason for removing this `test-linux-stable`?
This is permanent change or just temporary?

Comment (mordamax) [reply .gitlab-ci.yml:87]: temporary, just to run some pipeline

**Comment by chevdor** [comment]
Can you provide a description for this PR ? Is it a temp. test ?

**Comment by mordamax** [comment]
> Can you provide a description for this PR ? Is it a temp. test ?

Closed, sorry, forgot to follow up

**Pull request closed without merging**